### PR TITLE
Updated installation.rst file for Ubuntu

### PR DESCRIPTION
**Pull request by mitjap**

### CODE REVIEW

**Comment by mitar** [comment]
No. Please just add package names to the list and this is it.

`libxml2-dev` and `libxslt1-dev`.

**Comment by mitjap** [comment]
is it ok now?

File: docs/installation.rst [hunk 1]
```diff
@@ -111,3 +115,5 @@ The following Debian packages are needed:
 
 Be careful about required versions. It could be necessary to use packages from
 Debian testing or backports distribution.
+
+
```
Comment (mitar) [inline docs/installation.rst]: Unnecessary empty lines.

**Comment by mitar** [comment]
No, it is not OK. So in the first list, of prerequisites, list not with package names, but with project names and link to normal project page, not download pages.

And then under Debian section, add names of those additional packages.

So, please see how it is already done and try to do similar.

**Comment by mitjap** [comment]
Should there be an empty line at the end of file?

**Comment by mitar** [comment]
No.

**Comment by mitar** [comment]
So, no empty line. But newline yes.

**Pull request merged**